amf-api-contract/shared/src/main/scala/amf/apicontract/internal/spec/common/parser/ReferencesParsers.scala

```diff
@@ -19,10 +21,19 @@ case class WebApiRegister()(implicit ctx: WebApiContext) extends CollectionSideE
       case d: Module =>
         val library = ctx.declarations.getOrCreateLibrary(alias)
         d.declares.foreach(library += _)
+        collectExtensions(library, d)
       case fragment: Fragment => ctx.declarations += (alias -> fragment)
       case _                  => // ignore
     }
   }
+
+  private def collectExtensions(library: WebApiDeclarations, l: Module): Unit = {
+    l.references
```

Comment:
[CBI] - we have a filterType[T] helper to avoid the collect

Reply:
I'm interested, where?

Reply:
scala-common